backbone/app.py

```diff
@@ -3,39 +3,27 @@
 Run a rest API exposing the yolov5s object detection model
 """
 import io
-from flask import Flask, request
+from fastapi import FastAPI, File, UploadFile
+import uvicorn
 from PIL import Image
 from detect import Detector
 import os
 
-app = Flask(__name__)
+app = FastAPI(title="YOLOv5 Service")
 
 DETECTION_URL = "/v1/object-detection/yolov5s"
 
 detector = Detector()
-@app.route(os.path.join(DETECTION_URL,'detect'), methods=["POST"])
-def predict():
-    if not request.method == "POST":
-        return
 
-    if request.files.get("image"):
-        image_file = request.files["image"]
-        image_bytes = image_file.read()
 
-        img = Image.open(io.BytesIO(image_bytes))
-        results = detector.detect(img)
-        return results.to_json(orient="records")
+@app.post(os.path.join(DETECTION_URL, 'detect'))
+async def predict(image: UploadFile = File(...)):
 
-
-@app.route(os.path.join(DETECTION_URL,'set-model'), methods=["POST"])
-def set_model():
-    if not request.method == "POST":
-        return
-    model_path = request.form.get("model_path")
-    detector.set_model(model_path)
-    return "Model set"
+    contents = await image.read()
+    img = Image.open(io.BytesIO(contents))
+    results = detector.detect(img)
+    return results.to_json(orient="records")
 
 
 if __name__ == "__main__":
```

Comment:
Thank you for your hard-working please remove this line and with the help of the following code run unicorn directly.
https://github.com/virasad/semantic_segmentation_service/blob/b9cb50a9dc2802b206bfc262dd2d23216139156a/docker-compose.yml#L29